shm/cargo.cpp

```diff
@@ -9,3 +9,9 @@ Cargo& Cargo::operator-=(size_t amount) {
     amount_ -= amount;
     return *this;
 }
+
+bool Cargo::operator==(const Cargo& other) {
+    return (this->amount_ == other.amount_ &&
+            this->basePrice_ == other.basePrice_ &&
+            this->name_ == other.name_);
+}
```

Comment:
It can be without this->
But don't know which way is preferred by our group.

Reply:
Done.